### PR TITLE
Swish

### PR DESCRIPTION
**Pull request by raver119**
This PR adds new activation function: Swish

### CODE REVIEW

**Comment by maxpumperla** [comment]
so en vogue!

**Comment by raver119** [comment]
After reading 28 tweets about swish today - i gave up and opened laptop.

ср, 18 окт. 2017 г. в 18:26, Max Pumperla <notifications@github.com>:

File: include/ops/ops.h [hunk 1]
```diff
@@ -768,6 +768,30 @@ namespace simdOps {
...
+
+	template<typename T>
+	class SwishDerivative {
+	public:
+		no_op_exec_special
+		no_op_exec_special_cuda
+
+		op_def static T op(T d1, T *params) {
+			T ex = nd4j::math::nd4j_pow<T>(M_E, d1);
+			return (ex * (d1 + ex + 1)) / nd4j::math::nd4j_pow<T>((ex + 1) , (T)2.0f);
```
Comment (AlexDBlack) [inline include/ops/ops.h]: I can' reconcile this with the equation for f'(x) on page 2 - https://arxiv.org/pdf/1710.05941.pdf
Maybe I'm missing something, or it's just another form?

Comment (raver119) [reply include/ops/ops.h]: Merging as discussed in gitter.

**Pull request merged**